library/src/main/java/com/novoda/downloadmanager/lib/DownloadBatch.java

```diff
@@ -125,4 +149,77 @@ public boolean scanCompletedMediaIfReady(DownloadScanner downloadScanner) {
         return false;
     }
 
+    public boolean shouldShowActiveItem() {
+        int visibility = info.getVisibility();
+        return visibility == NotificationVisibility.ONLY_WHEN_ACTIVE
+                || visibility == NotificationVisibility.ACTIVE_OR_COMPLETE;
+    }
+
+    public boolean shouldShowCompletedItem() {
+        int visibility = info.getVisibility();
+        return visibility == NotificationVisibility.ONLY_WHEN_COMPLETE
+                || visibility == NotificationVisibility.ACTIVE_OR_COMPLETE;
+    }
+
+    public String getBigPictureUrl() {
+        return info.getBigPictureUrl();
+    }
+
+    public String getDescription() {
+        return info.getDescription();
+    }
+
+    public String getTitle() {
+        return info.getTitle();
+    }
+
+    public long getFirstDownloadBatchId() {
+        return downloads.get(0).getId();
+    }
+
+    public Statistics getLiveStatistics(LongSparseArray<Long> downloadSpeed) {
```

Comment:
by moving this to here, I noticed that the `DownloadBatch` already has `final` fields for `currentSizeBytes` and `totalSizeBytes` could these be used here? I'm not sure, didn't want to change that behaviour, but if anyone knows differently?